inst/tutorials/tidyverse_ws/tidyverse_ws.Rmd

```diff
@@ -0,0 +1,1067 @@
+
+Let's put it all together! Run the code below to produce your first ggplot:
+
+```{r, scatterplot, exercise = TRUE,  exercise.setup = "setup_long_dat"}
+dat %>%
+  ggplot(aes(x = Mean_O2, y = Mean_CO2)) +
+  geom_point()
+```
+
+Note that above R code has a warning. Warnings do not always indicate that there is an error in your code. For example, this warning tells us that there is missing data in our data frame, which we already know. 
+
+As you can see, there is a negative correlation between oxygen and carbon dioxide. In the code box below, plot the relationship between oxygen, on the x-axis, and depth, on the y-axis (`Depth`):
+
+```{r, scatterplot-2, exercise = TRUE,  exercise.setup = "setup_long_dat"}
+<data frame> %>%
+  ggplot(aes(<aesthetic maps>)) +
+  <geometric object>
+```
+
+```{r, scatterplot-2-solution}
+dat %>%
+  ggplot(aes(x = Mean_O2, y = Depth)) +
+  geom_point()
+```
+
+### Aesthetics
+
+What if we want to include all three variables (`Mean_O2`, `Mean_CO2`, and `Depth`) in the same two-dimensional plot? We ran out of axes after `x` and `y`, but we can use other aesthetic mappings to represent further variables. Aesthetic mappings include things like the `size`, the `shape`, or the `colour` of your points. 
+
+For instance, we can use the points' colour to represent depth. Run the code below and examine the output:
+
+```{r, scatterplot-color, exercise = TRUE,  exercise.setup = "setup_long_dat"}
+dat %>%
+  ggplot(aes(x = Mean_O2, y = Mean_CO2, colour = Depth)) +
+  geom_point()
+```
+
+(The `aes()` function accepts both `color` and `colour` as correct argument names.)
+
+As an alternative to point colour, we can also map a continuous variable by using differently sized points. Modify the box below so as to use the `size` aesthetic instead of the `colour` aesthetic:
+
+```{r, scatterplot-color-2, exercise = TRUE,  exercise.setup = "setup_long_dat"}
+dat %>%
+  ggplot(aes(x = Mean_O2, y = Mean_CO2, colour = Depth)) +
+  geom_point()
+```
+
+```{r, scatterplot-color-2-solution}
+dat %>%
+  ggplot(aes(x = Mean_O2, y = Mean_CO2, size = Depth)) +
+  geom_point()
+```
+
+We can also colour points by a categorical variable. Recall our `long_dat` data frame:
+
+```{r echo=FALSE, message=FALSE, warning=FALSE}
+dat <- select(geochemicals, Cruise, Date, Depth, starts_with("Mean_"))
+long_dat <- dat %>% 
+  pivot_longer(-c(Cruise, Date, Depth), 
+               names_to = "Chemical", 
+               values_to = "Concentration")
+long_dat
+```
+
+In the box below, `filter` this data frame so that it includes only oxygen and carbon dioxide. Then, plot the relationship between `Concentration` (in the x-axis) and `Depth` (in the y-axis), indicating the `Chemical` variable by point color:
```

Comment:
```suggestion
In the box below, `filter()` this data frame so that it includes only oxygen and carbon dioxide. Then, plot the relationship between `Concentration` (in the x-axis) and `Depth` (in the y-axis), indicating the `Chemical` variable by point color:
```